README.rst

```diff
@@ -372,11 +373,22 @@ Configure celery (you may use a different broker if you want):
 
     # here we show how to configure celery with redis but you can
     # use other brokers if you want, consult the celery docs
+    # Celery TIME_ZONE should be equal to django TIME_ZONE
+    # In order to schedule run_iperf_checks on the correct time intervals
+    CELERY_TIMEZONE = TIME_ZONE
     CELERY_BROKER_URL = 'redis://localhost/1'
     CELERY_BEAT_SCHEDULE = {
-        'run_checks': {
-            'task': 'openwisp_monitoring.check.tasks.run_checks',
-            'schedule': timedelta(minutes=5),
+       'run_checks': {
+        'task': 'openwisp_monitoring.check.tasks.run_checks',
+        # Executes only ping & config check every 5 min
+        'schedule': timedelta(minutes=5),
+        'args': (
+            [  # Checks path
+                'openwisp_monitoring.check.classes.Ping',
+                'openwisp_monitoring.check.classes.ConfigApplied',
+            ],
+        ),
+        'relative': True,
```

Comment:
aren't we missing the instructions to run the iperf check?

Reply:
I think it is mention [`here`](https://github.com/openwisp/openwisp-monitoring/tree/issue-385/iperf-check#3-configure-iperf-settings)